src/app/GitExtUtils/GitUI/Interops/RECT.cs

```diff
@@ -1,15 +1,58 @@
 ﻿using System.Runtime.InteropServices;
 
-namespace System;
-
-internal static partial class NativeMethods
+namespace System
```

Comment:
That's the opposite direction... We prefer namespaces without braces in new / heavily-revised files.

Reply:
Whoops! I was adding "Existing item" and completely forgot to select "As link".. it should've been like so
```
<Compile Include="..\GitUI\Interops\RECT.cs" Link="GitUI\Interops\RECT.cs" />
```